src/components/GameBoard.module.css

```diff
@@ -0,0 +1,30 @@
+.grid {
+  display: grid;
+  grid-template-columns: repeat(10, 1fr);
+  grid-template-rows: repeat(10, 1fr);
+  place-items: stretch;
+  place-content: center;
+  border-bottom: 1px solid #888;
+  border-right: 1px solid #888;
+}
+
+.cell {
+  place-self: stretch;
+  border-top: 1px solid #888;
+  border-left: 1px solid #888;
+}
+
+.unknown {
+  aspect-ratio: 1;
```

Comment:
Probably nicer when `aspect-ratio` is lifted in other class and then used in tandem with `classNames` or just pure JS: `<div className={[styles.aspectRatio, styles.waterBg].join(' ')}>`